com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Window.cs

```diff
@@ -137,18 +138,30 @@ public ConfigStyle(string label, string error, string button = resolve, MessageT
                 label: "Direct3D 12",
                 error: "Direct3D 12 is needed! (Editor restart is required)");
             public static readonly ConfigStyle dxrScreenSpaceShadow = new ConfigStyle(
-                label: "Screen Space Shadow",
-                error: "Screen Space Shadows are disabled in the current HDRP asset. You will not be able to toggle ray traced shadows on the lights in your scene. You can enable the feature in the HDRP asset under Lighting -> Shadows -> Screen Space Shadows", messageType: MessageType.Info);
+                label: "Screen Space Shadows (Asset)",
+                error: "Screen Space Shadows are disabled in the current HDRP asset. You will not be able to toggle ray traced shadows on the lights in your scene. You can enable the feature in the HDRP asset under Lighting > Shadows > Screen Space Shadows", messageType: MessageType.Warning);
```

Comment:
Screen Space Shadows are disabled in the current HDRP Asset which means you cannot enable ray-traced shadows for lights in your scene. To enable this feature, open your HDRP Asset, go to **Lighting > Shadows**, and enable **Screen Space Shadows**.